### ISSUE TITLE
There is a problem that only the 'pre' image remains.

### ISSUE THREAD

**Issue by JunhoSer** [ashnair1/xview2-toolkit#3]
I used this toolkit.
However, it was confirmed that only the "pre" image was left when the toolkit was executed.
I only want to get the "post" image, so I want you to tell me how to do it (Should I modify the xview_utils.py file, then how to do it?).

**Comment by ashnair1** [comment]
Can you give more details? Like the commands you ran and the output.

**Comment by JunhoSeo2075** [comment]
Figure 1 is the whole code I ran on colab.
![Q1](https://user-images.githubusercontent.com/65344591/175910194-42d31d4b-5cfc-4b1a-b255-48af502a74f4.png)

Figure 2 is the part that I modified in the py file (data->content, mini->test).
![Q2](https://user-images.githubusercontent.com/65344591/175910317-88984345-34b5-4595-94c1-d8254d23d428.png)

Figure 3 shows the code execution result. As described, only the "pre" image remains.
![Q3](https://user-images.githubusercontent.com/65344591/175910481-d4f09d80-24ce-4280-b044-a67eb4acc709.png)

Figure Q4 is the dataset used for the work. The dataset has a "post" image.
![Q4](https://user-images.githubusercontent.com/65344591/175911337-639a3ca4-9da8-4352-8020-f95c5dc0b090.png)

I was not good at coding, so I couldn't make proper corrections. I hope this problem will be solved.
